### PR TITLE
Release/1.4.0

### PR DESCRIPTION
**Pull request by malaffoon**
Hi Louis, Alex. This is the 1.4.0 update to the devops documentation (also includes User Guide and Admin Guide). I'll probably just merge this into `master` in a day or so but i wanted to give you a heads up. Feel free to drag Danny, Addie, Tuan, Heidi or whoever feels they need to review things. 

Note: the Admin Guide links to the devops documentation. But it does so in the `master` branch so those links won't work properly until i merge this PR.

### CODE REVIEW

File: deploy/admin-service.yml [hunk 1]
```diff
@@ -25,7 +25,7 @@ spec:
     spec:
       containers:
       - name: admin-service
-        image: smarterbalanced/rdw-reporting-admin-service:1.3.0-RELEASE
+        image: smarterbalanced/rdw-reporting-admin-service:1.4.0-RC17
```
Comment (malaffoon) [inline deploy/admin-service.yml]: hmm, i should change all these to RELEASE; will fix

File: docs/Runbook.SystemConfiguration.md [hunk 2]
```diff
@@ -135,6 +137,29 @@ reporting:
...
+
+#### Student Fields
+
+The system controls visibility into student demographic data for teachers. By default all student fields are available to all users. To change that behavior, modify properties in `rdw-reporting-service.yml`. Fields can be completely disabled, enabled for administrators but not teachers or enabled for all users including teachers.
+
+This example configuration is similar to California's policy: they use ELAS instead of LEP, nobody is allowed to see the students' economic disadvantage, and their teachers are restricted to just four fields (ELAS, migrant status, language, race/ethnicity):
+```yml
+reporting:
+  student-fields:
+    EconomicDisadvantage: disabled
```
Comment (malaffoon) [inline docs/Runbook.SystemConfiguration.md]: doh, need to fix case on these values

File: .gitignore [hunk 3]
```diff
@@ -1 +1,5 @@
 .idea/
+
+userguide/~\$porting-system-user-guide\.docx
```
Comment (esotrope) [inline .gitignore]: these must be the word produced temp docs huh

File: config/application.yml [hunk 4]
```diff
@@ -2,37 +2,249 @@
...
+    # The target reports display "Insufficient Data" when the standard error of the calculation is greater than 0.2.
+    # This is the default value provided by Smarter Balanced in the scoring calculation documentations and should
+    # only be changed if you really know why you are changing it.
+#    insufficient-data-cutoff: 0.2
+    # Target reports are only shown when there are a minimum number of students included in the report.
+    # Setting this to 0 means it will always be shown (even with a single student).
+#    min-number-of-students: 0
+
+server:
+  # the JWT can get pretty big ...
```
Comment (esotrope) [inline config/application.yml]: is it the JWT or the SAML xml?

**Pull request merged**